utility/google-services.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Same comments that were in the app module json apply here. Also add a comment above explaining why we need two json files.

Reply:
File deleted because of no usage.